eo-parser/src/main/resources/XMIR.xsd

```diff
@@ -64,7 +64,7 @@
       </xs:documentation>
     </xs:annotation>
     <xs:restriction base="xs:string">
-      <xs:pattern value="α[0-9]+|φ|[a-z]\S*|λ"/>
+      <xs:pattern value="α[0-9]+|φ|[a-z+]\S*|λ"/>
```

Comment:
@h1alexbel are you sure `[a-z+]` is the right pattern? Shouldn't it be `[a-z]+`?